src/main/java/HW13/MySQLConnection.java

```diff
@@ -0,0 +1,20 @@
+package HW13;
+
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class MySQLConnection {
+    private static final Logger LOGGER = Logger.getLogger(MySQLConnection.class);
+    private static final String URL = "jdbc:mysql://localhost:3306/new_schema?serverTimezone=UTC&useSSL=false";
```

Comment:
Please use property file for url, username and password. 
https://www.mkyong.com/java/java-properties-file-examples/